frontend/src/lib/utils.tsx

```diff
@@ -417,6 +417,22 @@ export function humanFriendlyNumber(d: number, precision: number = DEFAULT_DECIM
     return d.toLocaleString('en-US', { maximumFractionDigits: precision })
 }
 
+/** Format currency from string with commas and 2 decimal places. */
+export function humanFriendlyCurrency(d: string | undefined | number): string {
```

Comment:
I'd kind of expect this to return the currency symbol, `$`, since without that it's just a number, not a currency.

Reply:
That's a good point. I can add.